client/state/automated-transfer/reducer.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * External dependencies
+ */
+import { combineReducers } from 'redux';
+
+/**
+ * Internal dependencies
+ */
+import { AUTOMATED_TRANSFER_STATUS_SET } from 'state/action-types';
+import { automatedTransferStatusSchema } from './schema';
+import { createReducer } from 'state/utils';
+
+// Stores the status of automated transfers per site
+export const status = createReducer( {}, {
+	[ AUTOMATED_TRANSFER_STATUS_SET ]: ( state, action ) => ( {
+		...state,
+		[ action.siteId ]: action.automatedTransferStatus,
```

Comment:
this reducer doesn't actually give any indication to the kind of data we expect in the action. I guess that the schema does so explicitly and that's not too bad.

also I'm stil questioning what is best here. should we ever need to "set" the status? @nb recently gave the challenge to be suspicious about `SET` actions. what kinds of things might actually be happening which would trigger this? where does the status come from?

could we imagine something like this instead?

```js
[ AUTOMATED_TRANSFER_START ]: … [ action.siteId ]: 'start',
```

where we're trapping the action-centric messages to set the appropriate state?